packages/block-library/src/navigation-menu/editor.scss

```diff
@@ -12,7 +12,88 @@
 }
 
 .wp-block-navigation-menu-item {
-	.wp-block-navigation-menu-item {
+	.wp-block-navigation-menu-link {
 		margin-left: $grid-size-large;
 	}
 }
+
+/**
+ * Colors Selector component
+ */
+$colors-selector-size: 22px;
+.block-editor-block-colors-selector {
```

Comment:
Presumably, this all needs to be moved into a standalone component withint `block-editor/components`?

Reply:
Yes, but we agreed to do it in a next iteration. Probably pretty soon.